rts/c/values.h

```diff
@@ -6,6 +6,9 @@ typedef int (*writer)(FILE*, const void*);
 typedef int (*bin_reader)(void*);
 typedef int (*str_reader)(const char *, void*);
 
+FILE * OUTPUT;
```

Comment:
Make these `static`.

Also, all-uppercase identifiers are reserved for preprocessor constants. `output_file` and `output_file` would be better names, I think.

I still don't like the use of global variables for this at all, though.  Can't we do this in the entry point functions somehow?  I guess it's good enough for now.